scripts/enrich_rulings_ai.py

```
@@ -14,94 +15,139 @@
+                {"role": "system", "content": "You are a helpful assistant designed to output JSON."},
+                {"role": "user", "content": prompt},
+            ],
+            response_format="json",
+        )
+        content = response.choices[0].message.content
+        if isinstance(content, dict):
+            return content
+        return json.loads(content)
+    except Exception as e:
+        logging.error(f"Error calling OpenAI API: {e}")
+        return None
```

Ruff (0.11.9)

`error()` call on root logger

(LOG015)